static/src/js/actions/__tests__/authToken.test.js

```diff
@@ -3,6 +3,11 @@ import thunk from 'redux-thunk'
 import nock from 'nock'
 import * as tinyCookie from 'tiny-cookie'
 
+jest.mock('tiny-cookie', () => ({
```

Comment:
check if still needed

Reply:
these tests will fail if this is removed

Reply:
I'm still confused by why this is needed, but you can move this block to test-env.js and you don't have to mock the library in all the different tests, just once